src/routes/admin.rs

```diff
@@ -8,12 +8,20 @@ use rocket::{routes, Responder, Route, State};
 use sqlx::{Error, QueryBuilder, Row};
 
 pub fn admin_routes() -> Vec<Route> {
-    routes![
+    let mut routes = routes![
```

Comment:
These routes should not be under the payments feature, if there is something blocking those routes because of payments feature un-block it